contracts/Forwarder.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Any reason why we're bringing the entire contract solidity file into the repo?

Reply:
Just so we have everything for ABI generation (we don't actually need all, but figured for completeness, since we'll need to add more ABIs over time)